### DISCUSSION TITLE
Most setting options were lost suddenly and don't know why

### DISCUSSION THREAD

**Discussion by maotsuai** [ppy/osu#19428] [Q&A]
### Type

Game behaviour

### Bug description

I've played osu!lazer for time. Yesterday I imported a large amount of beatmaps from osu!stable (about 10,000) and played normally for a while. But when I open osu!lazer today, I found that the game was automatically taken into the initialization page, I tried to restart the game but it still goes the same. Then I finished the setup and went to the settings, only shrrtcut settings were still the same as before, other most other settings were lost.  

And another serendipitious bug is that when I quit playing from osu!std, game music will suddenly disapear, no matter how I tried, the music won't come back unless I restart the game. I've only encountered this bug few times.

I'm sorry for my bad English (

### Version

2022.723.0

### Logs

[performance.log](https://github.com/ppy/osu/files/9205085/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9205086/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9205087/updater.log)
[database.log](https://github.com/ppy/osu/files/9205088/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9205089/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9205090/network.log)

**Comment by peppy** [comment]
- You are running the game as administrator. Do not do this.

Everything else in your log files looks normal. For the music issue, we'd need to see a capture of it, and have logs from a session where that happens. For the "lost settings", can you share a few settings that you've noticed being reset?

**Reply by maotsuai** [reply]
I've reinstalled osu!lazer so I'm sorry for can't provide any screenshots. It occurs when I quit playing and return to the songs selecting screen, the whole display and sound effects were normal. I tried swithing songs back and forth and went back to the home screen to try  pausing and resuming the music, both didn't work.

For the lost settings, I found that language, skin, audio, tablet/mouse settings, HUD, offset and graphic settings were reset. My key configuration and online account were remained. I didn't change others.

**Reply by peppy** [reply]
The weird thing there is that skin and online account are stored to the same file. So I can't imagine how that can happen.

**Reply by maotsuai** [reply]
Sorry I didn't make my words clear, it's just the skin changing configuration, not the skin files.

**Reply by smoogipoo** [reply]
Do you have a `client_corrupt.realm` file or something similar in the osu! directory? Alternatively, can you provide a screenshot of the osu! directory? You can get to it by clicking the "open osu! directory" button in the in-game options.

**Reply by maotsuai** [reply]
Sorry it doesn't appear to have this file.
![屏幕截图 2022-07-28 134357](https://user-images.githubusercontent.com/81179261/181429575-6568edc5-ed7e-4ed8-a05d-6fb34af0dc5a.png)

**Reply by peppy** [reply]
To confirm, you didn't lose any beatmaps or skins that were already loaded in the game?

**Reply by maotsuai** [reply]
I checked them just now, the skin I used before works fine, so does the beatmaps. Maybe it's due to too many beatmaps imported at once?

**Reply by peppy** [reply]
Won't be able to provide any further help without logs (from exactly the session where it happens) sorry.
